src/decisionengine_modules/glideinwms/resource_dist_plugins.py

```diff
@@ -14,31 +8,35 @@
 }
 
 
-def order_resources(resources):
+def order_resources(resources, logger=None):
     ordered_resources = []
     rss_foms = pd.DataFrame()
 
     for rss, column_name in _RESOURCE_FROM_COLUMN_MAP.items():
         fom_df = resources.get(rss)
-        logger.info(f"Ordering resources based on {rss}")
+        if logger:
```

Comment:
I'm wondering if `if logger is not None:` is more explicit about what is going on here.  I go back and forth on the `__bool__` methods, but I suspect we care more if a logger is passed, not if it "is true"....

Reply:
A fair point--changes made accordingly.